src/stories/components/Disclaimer/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+
+import * as S from './styles';
+import { DisclaimerProps } from './types';
+
+export const Disclaimer: FC<DisclaimerProps> = ({ text = '' }) => {
+  return (
+    <S.Container data-testid="disclaimer" item xs={12}>
+      <span dangerouslySetInnerHTML={{ __html: text }} />
```

Comment:
Remover o uso do `dangerouslySetInnerHTML`. Deve ser evitado principalmente por questão de segurança mas também por questão de performance e manutenibilidade. Caso haja necessidade de renderizar elementos como `<br />`, acredito que uma alternativa melhor seria utilização de libs de Markdown.

```suggestion
      <span>{text}</span>
```

Reply:
Eu vou ainda além, acredito que para o boilerplate podemos remover o Disclaimer, tendo em vista que não foi solicitado para essa primeira versão